### PR TITLE
Add docs for 7 ops

### PR DESCRIPTION
**Pull request by PierceXiao413**
## Description ##
(Brief description on what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ x] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ x] Changes are complete (i.e. I finished coding on this PR)
- [x ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by reminisce** [comment]
Please rebase with upstream/numpy and add Examples section in the doc of each op.

File: python/mxnet/ndarray/numpy/_op.py [hunk 1]
```diff
@@ -501,20 +513,20 @@ def clip(a, a_min, a_max, out=None):
...
+    -----------
+    a : array_like
         Input array.
     axis1 : int
         First axis.
     axis2 : int
         Second axis.
 
-    Returns
+    Returns:
```
Comment (haojin2) [inline python/mxnet/ndarray/numpy/_op.py]: Pay attention to all extra whitespaces introduced.

File: python/mxnet/_numpy_op_doc.py [hunk 2]
```diff
@@ -21,10 +21,7 @@
 
 
 def _np_reshape(a, newshape, order='C'):
-    """
-    reshape(a, newshape, order='C')
-
-    Gives a new shape to an array without changing its data.
+    """Gives a new shape to an array without changing its data.
```
Comment (zoeygxy) [inline python/mxnet/_numpy_op_doc.py]: Please move the sentence to the next line.

File: python/mxnet/_numpy_op_doc.py [hunk 3]
```diff
@@ -176,3 +173,38 @@ def _np_cumsum(a, axis=None, dtype=None, out=None):
         `axis` is not None or `a` is a 1-d array.
     """
     pass
+
+def _np_broadcast_to(array, shape, subok=False):
+    r"""
+    _np_broadcast_to(array, shape)
```
Comment (zoeygxy) [inline python/mxnet/_numpy_op_doc.py]: For overriding the signature, you just need `broadcast_to(array, shape)`

File: python/mxnet/ndarray/numpy/_op.py [hunk 4]
```diff
@@ -503,20 +517,27 @@ def clip(a, a_min, a_max, out=None):
 def swapaxes(a, axis1, axis2):
     """Interchange two axes of an array.
 
-    Parameters
-    ----------
-    a : ndarray
+    Parameters:	
+    -----------
+    a : array_like
```
Comment (zoeygxy) [inline python/mxnet/ndarray/numpy/_op.py]: ndarray

File: python/mxnet/ndarray/numpy/_op.py [hunk 5]
```diff
@@ -503,20 +517,27 @@ def clip(a, a_min, a_max, out=None):
...
-    Returns
+    Returns:	
     -------
     a_swapped : ndarray
-        Swapped array. This is always a copy of the input array.
+
+    Examples:
+    >>> x = np.array([[1,2,3]])
+    >>> np.swapaxes(x,0,1)
+    array([[1],
```
Comment (zoeygxy) [inline python/mxnet/ndarray/numpy/_op.py]: My output:
```
>>> x = np.array([[1, 2, 3]])
>>> np.swapaxes(x, 0, 1)
array([[1.],
       [2.],
       [3.]])
```

File: python/mxnet/ndarray/numpy/_op.py [hunk 6]
```diff
@@ -882,3 +902,123 @@ def sqrt(x, out=None, **kwargs):
...
+
+    Inverse hyperbolic tangent element-wise.
+
+    Parameters:	
+
+    -----------
+    x : ndarray 
+        Input array.
+    out : ndarray, None, or tuple of ndarray and None.
+          A location into which the result is stored. If provided,
```
Comment (zoeygxy) [inline python/mxnet/ndarray/numpy/_op.py]: Please fix the `out` param part

File: python/mxnet/symbol/numpy/_symbol.py [hunk 7]
```diff
@@ -1555,4 +1600,127 @@ def sqrt(x, out=None, **kwargs):
...
+@set_module('mxnet.symbol.numpy')
+def fix(x, out=None):
+    """
+    Round to nearest integer towards zero.
+    
+    Round an array of floats element-wise to nearest integer towards zero. The rounded values are returned as floats.
+    
+    Parameters:	
+    ----------
+    x : Symbol or scalar
```
Comment (tingying2020) [inline python/mxnet/symbol/numpy/_symbol.py]: _Symbol

**Comment by haojin2** [comment]
@PierceXiao413 Seems like your commit history is not clean, please try to get rid of any commits not made by you.

File: python/mxnet/numpy/multiarray.py [hunk 8]
```diff
@@ -2057,3 +2035,128 @@ def sqrt(x, out=None, **kwargs):
...
+
+@set_module('mxnet.numpy')
+def arctanh(x, out=None, where=True, **kwargs):
+    r"""
+    arctanh(x, out=None, where=True)
+
+    Inverse hyperbolic tangent element-wise.
+
+    Parameters:	
+
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/numpy/multiarray.py]: fixed

File: python/mxnet/numpy/multiarray.py [hunk 9]
```diff
@@ -691,9 +645,29 @@ def pad(self, *args, **kwargs):
         raise AttributeError('mxnet.numpy.ndarray object has no attribute pad')
 
     def swapaxes(self, axis1, axis2):  # pylint: disable=arguments-differ
-        """Return a copy of the array with axis1 and axis2 interchanged.
-        Refer to `mxnet.numpy.swapaxes` for full documentation.
+        """Interchange two axes of an array.
+
+        Parameters:	
+        -----------
+        a : array_like
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/numpy/multiarray.py]: fixed

File: python/mxnet/ndarray/numpy/_op.py [hunk 10]
```diff
@@ -882,3 +902,123 @@ def sqrt(x, out=None, **kwargs):
     This function only supports input type of float.
     """
     return _unary_func_helper(x, _npi.sqrt, _np.sqrt, out=out, **kwargs)
+
+@set_module('mxnet.ndarray.numpy')
+def arctanh(x, out=None, where=True, **kwargs):
+    r"""
+    arctanh(x, out=None, where=True, **kwargs)
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: fixed

File: python/mxnet/ndarray/numpy/_op.py [hunk 11]
```diff
@@ -382,26 +381,41 @@ def multiply(x1, x2, out=None):
...
-        that the inputs broadcast to. If not provided or None, a freshly-allocated array
-        is returned.
+    Examples:
 
-    Returns
-    -------
-    out : ndarray or scalar
-        This is a scalar if both x1 and x2 are scalars.
+    >>> x = np.arange(5)
+    >>> np.true_divide(x, 4)
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: fixed

File: python/mxnet/ndarray/numpy/_op.py (hunk 6 above)
Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: ok fixed

File: python/mxnet/ndarray/numpy/_op.py (hunk 5 above)
Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: ok, fixed.

File: python/mxnet/ndarray/numpy/_op.py (hunk 4 above)
Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: fixed

File: python/mxnet/_numpy_op_doc.py (hunk 3 above)
Comment (PierceXiao413) [reply python/mxnet/_numpy_op_doc.py]: ok

File: python/mxnet/symbol/numpy/_symbol.py (hunk 7 above)
Comment (PierceXiao413) [reply python/mxnet/symbol/numpy/_symbol.py]: fixed

File: python/mxnet/symbol/numpy/_symbol.py [hunk 12]
```diff
@@ -1006,6 +1021,35 @@ def multiply(x1, x2, out=None):
...
+    x2 : symbolic or scalar 
+         Divisor array.
+    out : None or symbolic or scalar, optional
+          A location into which the result is stored. If provided, 
+          it must have a shape that the inputs broadcast to. 
+          If not provided or None, a freshly-allocated array is returned. 
+    where : symbolic or array, optional
+            Values of True indicate to calculate the ufunc at that position, 
+            values of False indicate to leave the value in the output alone.
+    **kwargs : For other keyword-only arguments, see the ufunc docs.
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/symbol/numpy/_symbol.py]: fixed

File: python/mxnet/symbol/numpy/_symbol.py [hunk 13]
```diff
@@ -1006,6 +1021,35 @@ def multiply(x1, x2, out=None):
...
+    """
+    Returns a true division of the inputs, element-wise.
+    Instead of the Python traditional ‘floor division’, 
+    this returns a true division. True division adjusts 
+    the output type to present the best answer, regardless 
+    of input types.
+
+    Parameters:	
+    ----------
+    x1 : symbolic or scalar
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/symbol/numpy/_symbol.py]: fixed

File: python/mxnet/numpy/multiarray.py [hunk 14]
```diff
@@ -2057,3 +2035,128 @@ def sqrt(x, out=None, **kwargs):
...
+          must have length equal to the number of outputs.
+    where : ndarray, optional
+            Values of True indicate to calculate the ufunc at that position, 
+            values of False indicate to leave the value in the output alone.
+    Returns:	
+    --------
+    out : ndarray or scalar
+          ndarray of the same shape as x. This is a scalar if x is a scalar.
+    Examples
+
```
[the remark this answers is not in the document]
Comment (PierceXiao413) [reply python/mxnet/numpy/multiarray.py]: ok, fixed.

File: python/mxnet/ndarray/numpy/_op.py [hunk 15]
```diff
@@ -882,3 +902,123 @@ def sqrt(x, out=None, **kwargs):
...
+    """
+
+    return _unary_func_helper(x, _npi.tan, _np.tan, out=out, **kwargs)
+
+@set_module('mxnet.ndarray.numpy')
+def fix(x, out=None):
+    r"""
+    Round an array of floats element-wise to nearest integer towards zero. 
+    The rounded values are returned as floats.
+
```
Comment (hgt312) [inline python/mxnet/ndarray/numpy/_op.py]: remove whitespaces

Comment (PierceXiao413) [reply python/mxnet/ndarray/numpy/_op.py]: ok

**Comment by haojin2** [comment]
Closing this PR now since the content is no longer needed.

**Pull request closed without merging**